src/Sentry/Internal/Hub.cs

```diff
@@ -219,6 +219,27 @@ public BaggageHeader GetBaggage()
         return propagationContext.GetOrCreateDynamicSamplingContext(_options).ToBaggageHeader();
     }
 
+    public TransactionContext ContinueTrace(
+        string? traceHeader,
+        string? baggageHeader,
+        string? name = null,
+        string? operation = null)
+    {
+        SentryTraceHeader? sentryTraceHeader = null;
+        if (traceHeader != null)
```

Comment:
we need a linter to help us move towards:

```suggestion
        if (traceHeader is not null)
```